lib/Doctrine/Common/NotifyPropertyChanged.php

```diff
@@ -2,19 +2,24 @@
 
 namespace Doctrine\Common;
 
-/**
- * Interface for classes that notify event listeners of changes to their managed properties.
- *
- * This interface is implemented by objects that manually want to notify their object manager or
- * other listeners when properties change, instead of relying on the object manager to compute
- * property changes itself when changes are to be persisted.
- */
-interface NotifyPropertyChanged
-{
+if (!interface_exists(\Doctrine\Persistence\NotifyPropertyChanged::class, false)) {
+    @trigger_error(
+        'The '.__NAMESPACE__.'\NotifyPropertyChanged class is deprecated since doctrine/persistence 1.2 and will be removed in 2.0.'
```

Comment:
```suggestion
        'The "'.__NAMESPACE__.'\NotifyPropertyChanged" interface is deprecated since doctrine/persistence 1.3 and will be removed in 2.0. Please use "..." instead.'
```

We could also use `sprintf` instead of string concatenation here.